docs/Dockerfile

```diff
@@ -1,86 +1,44 @@
 #
 # See the top level Makefile in https://github.com/docker/docker for usage.
 #
-FROM debian:jessie
+FROM docs/base:latest
```

Comment:
Hah -- I thought this was some strange hard-coded assumption, but no, you actually have control of this namespace: https://registry.hub.docker.com/u/docs/base/